frontend/components/Auction.vue

```diff
@@ -1,17 +1,21 @@
 <template>
     <TextBlock :title="`Auction #${auctionId}`">
-        <Alert v-if="auctionError" class="my-3" :message="auctionError" type="error" />
+        <div class="my-3">
+            <Alert v-if="auctionError && auctionError.showBanner" :message="auctionError.error" type="error" />
+        </div>
         <div v-if="auction">
-            <RestartBlock
-                v-if="auctionError === 'This auction is inactive and must be restarted'"
+            <AuctionRestartPanel
+                :disabled="!walletAddress"
```

Comment:
You're already passing `walletAddress` as a prop, so it's disabled state can be determined based on the `walletAddress`, which you already doing by the way. (but of course you can keep disabled prop in case we need to disable it even if there is a wallet)